skimage/restoration/tests/test_unwrap.py

```diff
@@ -37,8 +38,8 @@ def check_unwrap(image, mask=None):
     image_wrapped = np.angle(np.exp(1j * image))
     if mask is not None:
         print('Testing a masked image')
-        image = np.ma.array(image, mask=mask)
-        image_wrapped = np.ma.array(image_wrapped, mask=mask)
+        image = np.ma.array(image, mask=mask, fill_value=0)
```

Comment:
what happens when you pick a different fill value? zero is kinda a "corner case" what about 0.5?

Reply:
Sounds reasonable, fixed it!

Reply:
almost hehe, now I think it's done!